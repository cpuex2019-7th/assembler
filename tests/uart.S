    ;;  Initialization
    li t0, 0x44434241	
    li t1, 0x44434241
    li sp, 0x7F000000

    ;; read in bin and &hex
    sb t0, sp, 0
    sb t1, sp, 4    

    sh t0, sp, 8
    sh t1, sp, 16    

    sw t0, sp, 24
