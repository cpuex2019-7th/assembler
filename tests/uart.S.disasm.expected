lui t0, 279604
addi t0, t0, 577
lui t1, 279604
addi t1, t1, 577
lui sp, 520192
addi sp, sp, 0
sb t0, sp, 0
sb t1, sp, 4
sh t0, sp, 8
sh t1, sp, 16
sw t0, sp, 24
